redi-connect-admin/README.md

```diff
@@ -17,7 +17,7 @@
 1. Nuke old database
 2. Start mongod in mongodb-data folder: `cd mongodb-data && mongod --dbpath .`
 3. Run `yarn start` in `redi-connect-frontend`
-4. Run `./start-dev.sh` in `redi-connect-backend`
+4. Run `yarn start` in `redi-connect-backend`
```

Comment:
also thought of updating the docs now that the backend can run locally with `yarn start`. Kinda like when you can start all the projects with the same command. I think creates less overhead.

Reply:
`yarn` has been working for me without any problems, I havent used `npm` on the project at all and I was able to run everything.

Not a strong opinion from my side on neither of those, I just use yarn myself, but happy to discuss using npm. Point is to align on using one of those.